types/index.d.ts

```diff
@@ -0,0 +1,94 @@
+        encoding(encodings: string[]): string | string[] | false;
+        encoding(...encodings: string[]): string | string[] | false;
+
+        /**
+         * Return the first accepted encoding. If nothing in `encodings` is accepted, then `false` is returned.
+         * If no encodings are supplied, all accepted encodings are returned, in the order of the client's preference
+         * (most preferred first).
+         */
+        encodings(): string[];
+        encodings(encodings: string[]): string | string[] | false;
+        encodings(...encodings: string[]): string | string[] | false;
+
+        /**
+         * Return the first accepted language. If nothing in `languages` is accepted, then `false` is returned.
+         * If no languaes are supplied, all accepted languages are returned, in the order of the client's preference
+         * (most preferred first).
+         */
+        language(): string[];
+        language(languages: string[]): string | string[] | false;
+        language(...languages: string[]): string | string[] | false;
+
+        /**
+         * Return the first accepted language. If nothing in `languages` is accepted, then `false` is returned.
+         * If no languaes are supplied, all accepted languages are returned, in the order of the client's preference
+         * (most preferred first).
+         */
+        languages(): string[];
+        languages(languages: string[]): string | string[] | false;
+        languages(...languages: string[]): string | string[] | false;
+
+        /**
+         * Return the first accepted language. If nothing in `languages` is accepted, then `false` is returned.
+         * If no languaes are supplied, all accepted languages are returned, in the order of the client's preference
+         * (most preferred first).
+         */
+        lang(): string[];
+        lang(languages: string[]): string | string[] | false;
+        lang(...languages: string[]): string | string[] | false;
+
+        /**
+         * Return the first accepted language. If nothing in `languages` is accepted, then `false` is returned.
+         * If no languaes are supplied, all accepted languages are returned, in the order of the client's preference
+         * (most preferred first).
+         */
+        langs(): string[];
+        langs(languages: string[]): string | string[] | false;
+        langs(...languages: string[]): string | string[] | false;
+
+        /**
+         * Return the first accepted type (and it is returned as the same text as what appears in the `types` array). If nothing in `types` is accepted, then `false` is returned.
+         * If no types are supplied, return the entire set of acceptable types.
+         *
+         * The `types` array can contain full MIME types or file extensions. Any value that is not a full MIME types is passed to `require('mime-types').lookup`.
+         */
+        type(types: string[]): string[] | string | string[] | false;
```

Comment:
Does having `string[]` listed twice do something special or just a typo (sorry, only passiingly-familiar with type script)?

Reply:
sorry my fault. realized that the type in @types/accepts doesnt match what the library return. so i was curious if these types could be just shipped by the library itself.